analyzers/tests/SonarAnalyzer.Test/Extensions/SyntaxNodeExtensionsTest.cs

```diff
@@ -172,6 +173,56 @@ public void ArrowExpressionBody_WithNotExpectedNode_ReturnsNull()
         public void GetDeclarationTypeName_Struct() =>
             ExtensionsCS.GetDeclarationTypeName(SyntaxFactory.StructDeclaration("MyStruct")).Should().Be("struct");
 
+        [TestMethod]
+        public void GetBody_FieldDeclaration()
+        {
+            ExtensionsCS.GetBody(SyntaxFactory.FieldDeclaration(SyntaxFactory.VariableDeclaration(SyntaxFactory.ParseTypeName("int")))).Should().BeNull();
+        }
+
+        [TestMethod]
+        public void GetBody_PropertyDeclaration_SingleAccessor()
+        {
+            var code = "class AClass { int AProperty { get => 42; } }";
+            ExtensionsCS.GetBody(CSharpSyntaxTree.ParseText(code).Single<PropertyDeclarationSyntax>()).Should().BeNull();
```

Comment:
`ExtensionsCS.GetBody(CSharpSyntaxTree.ParseText(code).Single<T>` is repeated very often. Extract a helper
```cs
BlockSyntax ParseForBlockSyntax<T>(string code) where T: CSharpSyntaxNode
    => ExtensionsCS.GetBody(CSharpSyntaxTree.ParseText(code).Single<T>());
```